core/src/main/java/org/apache/iceberg/CachingCatalog.java

```diff
@@ -72,6 +182,8 @@ public Table loadTable(TableIdentifier ident) {
     TableIdentifier canonicalized = canonicalizeIdentifier(ident);
     Table cached = tableCache.getIfPresent(canonicalized);
     if (cached != null) {
+      // TODO - If this is a metadata table consider, checking that this is in sync
+      //        with the version that is presently cached. Could be confusing to user's though.
```

Comment:
This is probably fine.

Reply:
Will remove the comment then. Thanks for the input.